tests/mod.rs

```diff
@@ -2,12 +2,25 @@ extern crate pijama;
 
 use std::{panic, sync::mpsc, thread, time::Duration};
 
+use pijama::{
+    lir::Term,
+    machine::{arithmetic::CheckedArithmetic, MachineBuilder},
+    run_with_machine, LangResult,
+};
+
 mod ast;
 mod eval;
 mod parse;
 mod type_check;
 mod util;
 
+fn run(input: &str) -> LangResult<Term> {
+    let machine = MachineBuilder::default()
+        .with_arithmetic(CheckedArithmetic)
```

Comment:
Can we extract the default `MachineBuilder` to use in tests in its own function? So if we need some additional settings in some test we don't need to repeat all other options.